tico/passes/convert_conv_to_conv_transpose2d.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from torch.export import ExportedProgram
+
+from tico.serialize.circle_mapping import extract_shape
+from tico.utils import logging
+from tico.utils.graph import create_node
+from tico.utils.passes import PassBase, PassResult
+from tico.utils.trace_decorators import trace_graph_diff_on_pass
+from tico.utils.utils import is_target_node
+from tico.utils.validate_args_kwargs import ConvArgs
+
+
+@trace_graph_diff_on_pass
+class ConvertConvToConvTranspose2D(PassBase):
+    """
+    Convert 2-D transposed convolutions that were exported as the generic
+    `aten.convolution.default` into `aten.conv_transpose2d.input` calls.
```

Comment:
Have you tested we can keep `aten.conv_transpose2d.input` if we add it to `_preserve_ops` in `run_decompositions`?

Reply:
@jinevening Ah, yeah. I didn't know that. Let's just set it a preserved node.